apps/backend/src/services/compose_deployment.py

```
@@ -61,7 +61,7 @@ async def modify_compose_for_device(
         Returns:
             ComposeModificationResult with all modifications applied
         """
-        start_time = datetime.now(datetime.UTC)
+        start_time = datetime.now(timezone.utc)
```

Ruff (0.12.2)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)